corva/settings.py

```diff
@@ -1,15 +1,40 @@
 from os import getenv
-from typing import Final
+from typing import Optional
 
-API_ROOT_URL: Final[str] = getenv('API_ROOT_URL')
-DATA_API_ROOT_URL: Final[str] = getenv('DATA_API_ROOT_URL')
-APP_KEY: Final[str] = getenv('APP_KEY')
-APP_NAME: Final[str] = getenv('APP_NAME')
-API_KEY: Final[str] = getenv('API_KEY')
+from pydantic import BaseSettings
 
-# Logger
-LOG_LEVEL: Final[str] = getenv('LOG_LEVEL', 'WARN')
-LOG_ASSET_ID: Final[int] = int(getenv('LOG_ASSET_ID', -1))
 
-# Storage
-CACHE_URL: Final[str] = getenv('CACHE_URL')
+class CorvaSettings(BaseSettings):
```

Comment:
Let's change the name of this to Settings.

Reply:
Named it `CorvaSettings` to avoid vague imports like: `from corva.settings import Settings`, as pointed out in some previous PR. If you think it's alright, will return it

Reply:
If that's an issue, you can change the module name to `configuration.py`. I think that resolves everything?

Reply:
Yes, renamed the file, class and global variable